libstb/secvar/backend/edk2-compat.c

```diff
@@ -133,19 +133,11 @@ static int edk2_compat_process(void)
 	/* Perform a deep copy of the active bank to perform changes on.
 	 * Swap them at then end if processing succeeds */
 	list_head_init(&staging_bank);
-	list_for_each(&variable_bank, node, link) {
-		tmp = alloc_secvar(node->size);
-		tmp->flags = node->flags;
-		memcpy(tmp->var, node->var, tmp->size);
-		list_add_tail(&staging_bank, &tmp->link);
-	}
-
```

Comment:
Was this intended to be removed? I don't see anything replacing this, so this means that variables loaded (not empty ones allocated in `.pre_process()` are not going to be seen when we process updates